build/webpack.base.config.js

```diff
@@ -144,14 +146,27 @@ module.exports = {
                     },
                 ],
             },
+            ...(config.extractCss && config.isProd ? {
+                optimization: {
+                    minimizer: [
+                        new UglifyJsPlugin({
+                            cache: true,
+                            parallel: true,
+                            sourceMap: true,
+                        }),
+                        // Minimize extracted CSS files
+                        new OptimizeCSSAssetsPlugin({}),
```

Comment:
If you extract CSS files, you have to manually minimize them: https://github.com/webpack-contrib/mini-css-extract-plugin#minimizing-for-production

And then when you specify `optimization.minimizer`, you override the default for uglify so we re-specify that too